docs/project/governance.md

```diff
@@ -0,0 +1,96 @@
+# pyodide governance and decision-making
+
+The purpose of this document is to formalize the governance process used by the
+pyodide project, to clarify how decisions are made and how the various
+elements of our community interact.
+This document establishes a decision-making structure that takes into account
+feedback from all members of the community and strives to find consensus, while
+avoiding any deadlocks.
+
+Anyone with an
+interest in the project can join the community, contribute to the project
+design and participate in the decision making process. This document describes
+how that participation takes place and how to set about earning merit within
+the project community.
+
+## Roles And Responsibilities
+
+### Contributors
+
+Contributors are community members who contribute in concrete ways to the
+project. Anyone can become a contributor, and contributions can take many forms
+– not only code – as detailed in the {ref}`how_to_contibute`.
+
+### Community members team
+
+The community members team is composed of community members who have permission on
+Github to label and close issues. Their work is
+crucial to improve the communication in the project.
+
+Similarly to what has been decided in the [CPython project](
+https://devguide.python.org/triaging/#becoming-a-member-of-the-python-triage-team),
+any contributor may become a member of this team, after
+showing some continuity in participating in pyodide
+development (with pull requests and reviews).
```

Comment:
```suggestion
After participating in pyodide development with pull requests and reviews for a period of time, any contributor may become a member of the team.
The process for adding team members is modeled on the [CPython project](
https://devguide.python.org/triaging/#becoming-a-member-of-the-python-triage-team).
```

Reply:
Do we really want to focus on PRs and reviews here? Seems like we should also emphasize helping users in various ways (answering questions on gitter, responding to issues explaining how to do stuff).